app/serializers/project_serializer.rb

```diff
@@ -11,7 +11,6 @@ class ProjectSerializer < ActiveModel::Serializer
     latest_download_url
     latest_release_number
     latest_release_published_at
-    latest_stable_release
```

Comment:
@katzj I noticed you'd added the two attributes below [about two years ago](https://github.com/librariesio/libraries.io/pull/2247), so it seemed safe to remove `latest_stable_release`? LMK if this is a bad idea tho.